CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Bug fixes
 
+* Prevent SIGSEGV when passing NULL to bugsnag_set_user_env
```

Comment:
This one actually doesn't need a changelog entry since the bug hasn't been released

Reply:
The mazerunner scenario fails with a SIGSEGV on master too as there isn't a null check there either: https://github.com/bugsnag/bugsnag-android/blob/master/bugsnag-plugin-android-ndk/src/main/jni/bugsnag.c#L140